classifier.py

```diff
@@ -3574,49 +3613,29 @@ def target_age(self, audience):
             return Classifier.default_target_age_for_audience(audience)
 
         # Only consider the most reliable classifications.
-        reliable_classifications = self.most_reliable_target_age_subset
-
+        
         # Try to reach consensus on the lower and upper bounds of the
         # age range.
-        target_age_mins = []
```

Comment:
We already built the Counters for target age min and max, so this code is no longer needed.